LLMonFHIR/FHIR Views/InspectResourceView.swift

```diff
@@ -0,0 +1,141 @@
+//
+// This source file is part of the Stanford Spezi project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziFHIR
+import SpeziLLM
+import SpeziViews
+import SwiftUI
+
+struct InspectResourceView: View {
+    @Environment(FHIRResourceInterpreter.self) var fhirResourceInterpreter
+    @Environment(FHIRResourceSummary.self) var fhirResourceSummary
+    
+    @State var interpreting: ViewState = .idle
+    @State var loadingSummary: ViewState = .idle
+    
+    var resource: FHIRResource
+    
+    
+    var body: some View {
+        List {
+            summarySection
+            interpretationSection
+            resourceSection
+        }
+            .navigationTitle(resource.displayName)
+            .viewStateAlert(state: $interpreting)
+            .viewStateAlert(state: $loadingSummary)
+            .task {
+                #if !TEST
+//                interpret()
+                #endif
+            }
```

Comment:
We might want to go though the elements like this and ensure that we remove uncommented code & other elements in the long run.